src/Symfony/Component/Workflow/Workflow.php

```diff
@@ -1,26 +1,17 @@
 <?php
 
-/*
```

Comment:
Sorry, I've been moving code around in different classes and created new ones.. Many small issues like this on my part in this PR. I will make sure to clean it up tonight.